### PR TITLE
Update configure-exclusions-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more reference to the list-of-automatic-exclusions important note in the document

fixes #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10316

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-exclusions-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ To configure and validate exclusions, see the following:
 ## Recommendations for defining exclusions
 
 > [!IMPORTANT]
-> Microsoft Defender Antivirus includes many automatic exclusions based on known operating system behaviors and typical management files, such as those used in enterprise management, database management, and other enterprise scenarios and situations.
+> Microsoft Defender Antivirus includes many automatic exclusions based on known operating system behaviors and typical management files, such as those used in enterprise management, database management, and other enterprise scenarios and situations. Refer to the [list of automatic exclusions](/configure-exclusions-microsoft-defender-antivirus#the-list-of-automatic-exclusions)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-exclusions-microsoft-defender-antivirus.md]:
```suggestion
> Microsoft Defender Antivirus includes many automatic exclusions based on known operating system behaviors and typical management files, such as those used in enterprise management, database management, and other enterprise scenarios and situations. For more information, refer to the [list of automatic exclusions](/configure-exclusions-microsoft-defender-antivirus#the-list-of-automatic-exclusions).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**